dashboard/app/views/layouts/_hotjar.html.haml

```diff
@@ -0,0 +1,10 @@
+-# Code snippet needed for Hotjar heatmap tracking.
```

Comment:
Could you link to the documentation for this added code?

Reply:
https://help.hotjar.com/hc/en-us/articles/115011639927-What-is-the-Hotjar-Tracking-Code